software/datasette.yml

```
@@ -1,6 +1,6 @@
 name: Datasette
 website_url: https://datasette.io/
-description: An open source multi-tool for exploring and publishing data, easy import and export and database management.
+description: Explore and publish data, easy import and export and database management.
```

Sentence here is incomplete. I'd suggest changing it to
```suggestion
description: A tool to explore and publish data, easy import and export and database management.
```